pkg/cmd/step/create/test_data/step_create_task/packs/jx-syntax-in-buildpack/pipeline.yaml

```diff
@@ -0,0 +1,28 @@
+agent:
+  label: jenkins-maven
+  container: maven
```

Comment:
I find it so confusing that we are dealing with 'agent', 'container', 'image'. I guess some things derive from trying to consolidate different models, but would it still not possible to have a single concept in the `jenkins-x.yml`? But maybe this ship has sailed?